services/platform/platform.go

```
@@ -586,62 +336,30 @@ func (s *Service) GetContactInformation(ctx context.Context, req *platformpb.Get
 		return nil, status.Error(codes.Aborted, "PMM server is not connected to Portal")
 	}
 
-	endpoint := fmt.Sprintf("https://%s/v1/orgs/%s", s.host, ssoDetails.OrganizationID)
+	resp, err := s.client.GetContactInformation(ctx, accessToken, ssoDetails.OrganizationID)
```

error is ignored here